actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -41,7 +41,7 @@ class Request
         HTTP_ACCEPT_LANGUAGE HTTP_CACHE_CONTROL HTTP_FROM
         HTTP_NEGOTIATE HTTP_PRAGMA HTTP_CLIENT_IP
         HTTP_X_FORWARDED_FOR HTTP_ORIGIN HTTP_VERSION
-        HTTP_X_CSRF_TOKEN HTTP_X_REQUEST_ID HTTP_X_FORWARDED_HOST
+        HTTP_X_CSRF_TOKEN HTTP_X_FORWARDED_HOST
```

Comment:
Breaks compatibility. Still want this method for the default case.

Reply:
Fair, added it back